
Header header
float64 travedDist
float64 timeConsumed